mava/utils/make_env.py

```diff
@@ -90,6 +92,18 @@ def add_extra_wrappers(
     # Disable the AgentID wrapper if the environment has implicit agent IDs.
     config.system.add_agent_id = config.system.add_agent_id & (~config.env.implicit_agent_id)
 
+    if config.system.add_graph_wrapper:
+        if config.env.env_name == "MPE":
+            assert isinstance(train_env, MPEWrapper) and isinstance(
+                eval_env, MPEWrapper
+            ), "Attempting to add mpe graph wrapper to non-MPE environment"
+
+            train_env = MPEGraphWrapper(train_env)
+            eval_env = MPEGraphWrapper(eval_env)
+        else:
+            train_env = GraphWrapper(train_env)
+            eval_env = GraphWrapper(eval_env)
```

Comment:
Can we add this to the registry instead otherwise every time we make a new graph wrapper we'll need to add to this if statement?

It'll require a slight rework to make other registries like the jumanji registry. So jaxmarl etc would become a dict like this:
```
{env_name: {"wrapper": WrapperClass, ...}}
```

where ... is optional extra fields - in this case only MPE would have `"graph_wrapper": MPEGraphWrapper`. This would then allow us to always get the graph wrapper like this: `graph_wrapper = _[...]_registry[env_name].get("graph_wrapper", GraphWrapper)`